flexget/plugins/urlrewrite_divxatope.py

```diff
@@ -36,7 +36,15 @@ def parse_download_page(self, url):
             download_link = soup.findAll(href=re.compile('redirect|redirectlink'))
             download_href = download_link[0]['href']
             if "url" in download_href:
-                return download_href[download_href.index('url=') + 4:]
+                redirect_search = re.search('.*url=(.*)', download_href)
+                if redirect_search:
+                        redirect_link = redirect_search.group(1)
+                else:
+                    raise UrlRewritingError('Redirect link for %s could not be found %s' % url)
+                if redirect_link.startswith("http"):
+                    return redirect_link
+                else:
+                    return "http://www.divxatope.com/" + redirect_link
```

Comment:
We try to standardize on single quotes `'`, can we switch all the strings to that?